docs/markdown/Tutorial.md

```diff
@@ -141,29 +147,29 @@ int main(int argc, char **argv)
   GtkApplication *app;
   int status;
 
-  app = gtk_application_new(NULL, G_APPLICATION_DEFAULT_FLAGS);
+  app = gtk_application_new("org.gtk.example", G_APPLICATION_DEFAULT_FLAGS);
   g_signal_connect(app, "activate", G_CALLBACK(activate), NULL);
   status = g_application_run(G_APPLICATION(app), argc, argv);
-  g_object_unref(app);
+  g_object_unref(app);  // Free from memory when program terminates
 
   return status;
-} // end of function main
+}
 ```
 
 Then we edit the Meson file, instructing it to find and use the GTK+
 libraries.
 
 ```meson
 project('tutorial', 'c')
-gtkdep = dependency('gtk+-3.0')
+gtkdep = dependency('gtk4')
 executable('demo', 'main.c', dependencies : gtkdep)
 ```
 
 If your app needs to use multiple libraries, you need to use separate
 [[dependency]] calls for each, like so:
 
 ```meson
-gtkdeps = [dependency('gtk+-3.0'), dependency('gtksourceview-3.0')]
+gtkdep = [dependency('gtk4'), dependency('gtksourceview-4')]
```

Comment:
I find it is more idiomatic in meson if you split these up.

```
gtk_dep = ...
gtksourceview_dep = ...
```

Also, I am pretty sure gtksourceview-5 is the pkgconfig file name.